lib/pos_staking_rewards.go

```diff
@@ -225,9 +225,9 @@ func (bav *UtxoView) distributeStakingReward(
 				IsValidatorCommission: isValidatorCommission,
 			},
 		}
-		stakeEntry.StakeAmountNanos = uint256.NewInt(0).Add(stakeEntry.StakeAmountNanos, uint256.NewInt(rewardNanos))
```

Comment:
In PR #1163, we have a change in `lib/pos_snapshot_entries.go` - it is related to `SnapshotValidatorSetTotalStakeAmountNanos` so it's expected that we won't see this here as logic related to this was deleted.